charts/c100-application/values.yaml

```diff
@@ -45,3 +45,7 @@ base:
           alias: MAINTENANCE_ALLOWED_IPS
 redis:
   enabled: false
+useInterpodAntiAffinity: true
```

Comment:
this is already enabled by default:
https://github.com/hmcts/chart-java/blob/8e0c2fcdac5ef416636e38e328ecb4de111f2d22/java/values.yaml#L32

Reply:
Without adding this here, "hmcts.interpodantiaffinity.v2" doesn't load.